pom.xml

```diff
@@ -150,24 +149,6 @@
         <artifactId>animal-sniffer-annotations</artifactId>
         <version>${animal.sniffer.version}</version>
       </dependency>
-      <dependency>
```

Comment:
we no longer support targeting java < 1.8
(and do not yet support targeting java > 1.8)